### PR TITLE
SOC-2409 | Post Card Redesign

### PR DESCRIPTION
**Pull request by bkoval**
Work In Progress

## Links

* http://wikia-inc.atlassian.net/browse/SOC-2409

## Reviewers
@kvas-damian

### CODE REVIEW

File: front/main/app/styles/module/_discussion.scss [hunk 1]
```diff
@@ -1,10 +1,15 @@
 $horizontal-padding: 18px;
 $post-card-right-icon-margin: -9px;
 $post-card-top-note-height: 40px;
-$post-details-content-line-height: 1.4rem;
+$mobile-content-font-size: $type-smallest;
+$mobile-content-line-height: $line-height-min;
+$desktop-content-font-size: rem-calc(16);
+$desktop-content-line-height: rem-calc(24);
```
Comment (kvas-damian) [inline front/main/app/styles/module/_discussion.scss]: Can we try using already defined variables like: $type-small ?

File: front/main/app/styles/module/_discussion.scss [hunk 2]
```diff
@@ -574,37 +626,48 @@ body.discussions {
...
+					bottom: 0;
+					left: 0;
 				}
 				width: $post-details-outer-width;
 
 				&.show-top-note {
 					padding-top: 0;
 				}
 
+				.side-spaced {
```
Comment (kvas-damian) [inline front/main/app/styles/module/_discussion.scss]: It looks like some kind of centering of content. Is it possible to use different approach where browser calculates what is the right position for element to be centered?

Comment (bkoval) [inline front/main/app/styles/module/_discussion.scss]: It probably is possible. However I don't want to change that logic now (I just moved it to a different class) as I would like to cut scope of this ticket to deliver it quickly. Can we add it to AC in https://wikia-inc.atlassian.net/browse/SOC-2530?

**Comment by kvas-damian** [comment]
@bkoval Please download translations

File: front/main/app/styles/module/_discussion.scss (hunk 2 above)
Comment (kvas-damian) [inline front/main/app/styles/module/_discussion.scss]: Ok, please add it there.
BTW :abc:

File: front/main/app/symbols/main/upvote-reply.svg [hunk 3]
```diff
@@ -1,3 +1 @@
-<svg width="10" height="8" viewBox="0 0 10 8" xmlns="http://www.w3.org/2000/svg">
-  <path d="M4.69 1.412L1.528 4.574 1 4.046 5.046 0l4.047 4.046-.528.528-3.167-3.167V7.5h-.704L4.69 1.412z"/>
-</svg>
+<svg width="10" height="10" viewBox="0 0 10 10" xmlns="http://www.w3.org/2000/svg"><g fill-rule="evenodd"><path class="upvote-icon-background" d="M-4-4h18v18H-4z"/><path d="M9.707 4.292l-4-4C5.615.2 5.504.127 5.382.076c-.244-.1-.52-.1-.764 0-.123.05-.233.124-.325.216l-4 4c-.39.39-.39 1.023 0 1.414C.488 5.9.743 6 1 6c.256 0 .512-.1.707-.294L4 3.413V9c0 .55.448 1 1 1s1-.45 1-1V3.412l2.293 2.293C8.488 5.9 8.743 6 9 6c.256 0 .512-.1.707-.294.39-.39.39-1.023 0-1.414"/></g></svg>
```
Comment (kvas-damian) [inline front/main/app/symbols/main/upvote-reply.svg]: Do we need classnames in svgs?

File: front/main/app/symbols/main/reply.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Reply-24x24</title><g fill="none" fill-rule="evenodd"><path d="M0 0h24v24H0z"/><path d="M21 11.273c0 4.01-4.037 7.272-9 7.272-.752 0-1.508-.078-2.252-.233l-4.22 1.636c-.09.035-.184.052-.278.052-.162 0-.322-.05-.454-.148-.21-.155-.32-.405-.292-.66l.396-3.45C3.672 14.47 3 12.896 3 11.273 3 7.263 7.037 4 12 4s9 3.263 9 7.273z" fill="#999"/></g></svg>
```
Comment (kvas-damian) [inline front/main/app/symbols/main/reply.svg]: Please remove `title` tag.

File: front/main/app/styles/module/_discussion.scss [hunk 5]
```diff
@@ -573,37 +642,48 @@ body.discussions {
...
 				border-bottom: 1px solid $discussion-reply-border-color;
 				margin: auto;
 				padding: {
 					top: $post-details-post-vertical-padding;
-					right: $horizontal-padding + ($post-details-outer-width - $post-details-post-width) / 2;
-					bottom: $post-details-post-vertical-padding + $post-details-post-margin-bottom;
-					left: $horizontal-padding + ($post-details-outer-width - $post-details-post-width) / 2;
+					right: 0;
+					bottom: 0;
+					left: 0;
```
Comment (kvas-damian) [inline front/main/app/styles/module/_discussion.scss]: :abc:

File: front/main/app/symbols/main/upvote-reply.svg (hunk 3 above)
Comment (bkoval) [inline front/main/app/symbols/main/upvote-reply.svg]: Yeah, unfortunately this is needed for setting the transparent background on the SVG :)

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**